style.css

```diff
@@ -0,0 +1,200 @@
+@import url('https://fonts.googleapis.com/css?family=Nunito:400,400i,700');
+/* Eliminate bullet point, spread navigation on a row */
+
+body {
+    font-family: 'Nunito', sans-serif;
+}
+.nav {
+    display: grid;
+    grid-template-columns: 60% 30% 10%;
+    list-style-type: none;
+    font-weight: 700;
+   
+}
+.nav li:hover {
+    color: blue;
+}
+
+/* Create three columns to store each element of main-page */
+.intro {
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+    text-align: center ;
+}
+
+.intro h1 {
+    grid-row: 3/4;
+    grid-column: 4/5;  
+}
+.intro sub {
+    grid-row: 4/5;
+    grid-column: auto/span 7;
+    display: inline-block;
+    align-items: center;
+    font-size: 1.5em;
+    font-weight: bold;
+}
+
+.intro a {
+    grid-row: 5/6;
+    grid-column: 4/5;
+    font-size: 1.25em;
+    text-decoration: none;
+    color: black;
+    align-self: center;
+    border: solid lightgrey;
+    border-radius: 10px;
+    padding: 10px;
+    
+}
+
+.intro a:hover {
+    color: blue;
+}
+
+hr {
+    border-top: 1px solid #8c8b8b;
+    margin-top: 3%;
```

Comment:
Cool, didn't know you could use percentages as measurements here. Smart, makes it more flexible/scalable